src/PlayerRegistration.ts

```diff
@@ -2,4 +2,5 @@
 
 export const PlayerRegistrationUrls: any[] = [
 //  "http://localhost:3000",
+    "http://159.203.156.37/player/register"
```

Comment:
GETting this in my browser returned `Cannot GET /player/register` (12:20 AM Sunday morning)